src/components/apphost.js

```diff
@@ -304,19 +304,26 @@ function askForExit() {
         return;
     }
 
-    import('../components/actionSheet/actionSheet').then((actionsheet) => {
-        exitPromise = actionsheet.show({
-            title: globalize.translate('MessageConfirmAppExit'),
-            items: [
-                { id: 'yes', name: globalize.translate('Yes') },
-                { id: 'no', name: globalize.translate('No') }
-            ]
-        }).then(function (value) {
-            if (value === 'yes') {
-                doExit();
-            }
-        }).finally(function () {
-            exitPromise = null;
+    import('../utils/dashboard').then(() => {
```

Comment:
At least use the result.
```suggestion
    import('../utils/dashboard').then((Dashboard) => {
```
Should we use camelCase (`dashboard`) since it is a variable? :thinking:

Reply:
`utils/dashboard` exports `Dashboards` so it is available for use without grabbing the result.

Tested on TV and working as intended.